usecases/schema/parser.go

```diff
@@ -197,12 +197,8 @@ func (p *Parser) parseTargetVectorsVectorIndexConfig(class *models.Class) error
 		if err != nil {
 			return fmt.Errorf("parse vector config for %s: %w", targetVector, err)
 		}
-		if vectorizerModuleName != "none" && parsed.IsMultiVector() != isMultiVector {
-			if isMultiVector {
-				return fmt.Errorf("parse vector config for %s: configured vectorizer: %q supports multi vectors but multi vector index is not configured", targetVector, vectorizerModuleName)
-			} else {
-				return fmt.Errorf("parse vector config for %s: multi vector index configured but vectorizer: %q doesn't support multi vectors", targetVector, vectorizerModuleName)
-			}
+		if vectorizerModuleName != "none" && !isMultiVector && parsed.IsMultiVector() != isMultiVector {
```

Comment:
nit, i think the suggestion i've made below is more clear now that we only error if the index is multi vector but the vectorizer does not support multi vector. not blocking though, feel free to make this change or leave it as is

```suggestion
		if parsed.IsMultiVector() && vectorizerModuleName != "none" && !isMultiVector {
```